Formula/s/sshx.rb

```diff
@@ -0,0 +1,28 @@
+class Sshx < Formula
+  desc "Fast, collaborative live terminal sharing over the web"
+  homepage "https://sshx.io"
+  url "https://github.com/ekzhang/sshx/archive/refs/tags/v0.3.1.tar.gz"
+  sha256 "3bc2abc59be26c9eefaf7fc999aa4a3723d2916414c319fd06555415278acb64"
+  license "MIT"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
```

Comment:
actually not livecheck, but version check in the test block, sorry for not being clear (default livecheck works fine)

```suggestion
```

Reply:
No worries, I believe I added what you're looking for to the test block